spec/features/users/new_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+
+RSpec.describe "user registration page", type: :feature do
+  before(:each) do
+    @user1 = User.create!(name: "boston", email: "boston@example.com")
+    visit "/register"
+  end
+
+  describe "when I visit the user registration page" do
+    describe "Happy Path" do
```

Comment:
```suggestion
    context "Happy Path" do
```

This would be a good use case for "context" instead of "describe".